declarations/Perplexity.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Perplexity",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.perplexity.ai/hub/legal/terms-of-service",
+      "select": [
+        "[name=Post]"
+      ],
+      "executeClientScripts": true
+    },
+    "Privacy Policy": {
+      "fetch": "https://www.perplexity.ai/hub/legal/privacy-policy",
+      "select": [
+        "[name=Post]"
+      ],
+      "executeClientScripts": true
+    },
+    "Developer Terms": {
+      "fetch": "https://www.perplexity.ai/hub/legal/perplexity-api-terms-of-service",
+      "select": [
+        "[name=Post]"
+      ],
+      "executeClientScripts": true
+    },
+    "Business Privacy Policy": {
```

Comment:
In my opinion, this document is not a `Business Privacy Policy`. I think it's a document that specifies the treatment of personal data when users interact programmatically with the service. So we're maybe talking about a new type, which could be called, for example, `Developer Privacy Policy`. If you agree with my analysis, I suggest that you delete the tracking of `Business Privacy Policy` and I will open a [discussion](https://github.com/OpenTermsArchive/terms-types/discussions) to add this new type of terms.

Reply:
Agree that it fits better but "Developer privacy policy" sounds like it's about the privacy of the developers. Why not introduce "API privacy policy" and "API terms and conditions"? Surely that's a pretty common use case?

Reply:
The [discussion](https://github.com/OpenTermsArchive/terms-types/discussions/29) I've just opened will enable us to check that these are common cases. We integrate new types of terms when we have been able to reference 3 documents of the same type and we have found a consensus name.